.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
According to the project requirement, you were asked to add `test.md` file to the `.gitignore` file, just the same way you added the `node_modules` as this is a practise to enlighten you on how to ignore some files on the remote Github repository 😄 

![test md](https://user-images.githubusercontent.com/20537712/234672599-409bcf7e-8e7a-48e7-8d4b-12a09926f140.png)